apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -161,23 +182,30 @@ def simulate_game(
     states_dict: Dict,
     horizon: int = 5,
 ):
+    """Simulates a game from a given state.
+
+    Args:
+        model: The model to use for the simulation.
+        state (torch.Tensor): The initial state.
+        t_time (int): The current time step.
+        max_steps (int): The maximum number of steps to simulate.
+        game_tree (Dict): The game tree.
+        states_dict (Dict): The states dictionary.
+        horizon (int): The horizon to use for the simulation.
+    """
     idx = t_time
     max_steps = min(max_steps, t_time + horizon)
     state_hash = to_hash(extract_present_state(state))
     trajectory = []
     # selection
     while state_hash in game_tree:
         (
-            (state_tensor, full_actions),
-            old_idx_to_new_idx,
+            possible_states,
```

Comment:
please keep the old code modification (the code in red should replace the code in green)